Controllers/MoviesController.cs

```diff
@@ -51,6 +51,22 @@ public IActionResult Edit(int id, Movie movie)
             return View(movie);
         }
 
+        public IActionResult Create()
+        {
+            return View();
```

Comment:
return some ViewData for use create or edit on form an return to Edit.cshtml view